gammapy/scripts/jupyter.py

```diff
@@ -0,0 +1,166 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""Command line tool to perform actions on jupyter notebooks."""
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+import click
+import logging
+import subprocess
+import sys
+import time
+
+log = logging.getLogger(__name__)
+
+try:
+    import nbformat
```

Comment:
You can't have this at the top of the module, it needs to be a delayed import in the functions that use this.
See https://ci.appveyor.com/project/cdeil/gammapy/build/1.0.4455/job/qfhona74oc6yy5sw#L1569

I would just let the ImportError occur. We do this even for users e.g. in functions that use scipy or matplotlib in Gammapy, so even more so for dev tools I think it's OK to see an ImportError occur.

Reply:
What do you mean by a 'delayed import'? Am I supposed to leave the imports even if CI fails? I was struggling to fix the CI by any means.

Reply:
With delayed import I mean to put the import into the function where you use it on the first line. I don't know if there's a proper term for that.

Then CI and tests will not fail if they just import the module.
And when writing tests for code that uses an optional dependency, we have the `requires_dependency` decorator that we use to mark each such test function.